model/ft_sensor_gamma.urdf.xacro

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
```

Comment:
What is this file for?

The gamma model is already included in the https://github.com/CentroEPiaggio/force-torque-sensor/blob/master/model/models.urdf.xacro

If you want to have an examble of the gamma instatiated, it should be placed in the `robot` folder, not in the `model`.

Just to keep the structure.